core/node/da_dispatcher/src/lib.rs

```diff
@@ -0,0 +1,4 @@
+pub use self::da_dispatcher::DataAvailabilityDispatcher;
+
+pub mod da_dispatcher;
```

Comment:
Nit: Why does this module need to be public? IIUC, it only has `DataAvailabilityDispatcher`, which re-exported anyway.

Reply:
removed pub modifier